### DISCUSSION TITLE
Magnon band structure negative energy

### DISCUSSION THREAD

**Discussion by lisurui6** [mailhexu/TB2J#31] [Q&A]
Following the TB2J_examples of Wannier/FeO_QE_Wannier90 and SrMnO3_QE_Wannier90, I tried to generate the magnon band structures for each, using TB2J_magnon.py script. However the energies of the bands are negative (see attached). 

Do you know why?

Thanks

FeO
![image](https://github.com/mailhexu/TB2J/assets/17573510/47607307-bdc8-48af-aace-d95699539155)

SrMnO3
![image](https://github.com/mailhexu/TB2J/assets/17573510/39cb2731-1c7d-448e-baf9-32c9da6ff70d)

**Comment by mailhexu** [comment]
Hi, 
This is because the current version of TB2J take the FM state as energy "0". Negative energy means that that the corresponding state has lower energy than the FM state. 
 For FeO, the two Gamma point band are FM (0meV) and AFM states(negative). And for SrMnO3, the lowest energy at R is AFM. 
Best wishes,
HeXu
